viewer/Viewer.cpp

```diff
@@ -173,6 +250,25 @@ namespace Enki
 		doDumpFrames = doDump;
 	}
 	
+	void ViewerWidget::toogleTrackball()
+	{
+		trackballView = !trackballView;
+		camera.radius = 20;
+	}
+
+	void ViewerWidget::sendMessage(QString msg, unsigned int persistance)
+	{
+		messageList.push_back(viewerMessage(msg,persistance));
+	}
+
+	void ViewerWidget::showHelp()
+	{
+		QString str(controlHelp);
+		QStringList slist = str.split("\n");
+		for(int i=0; i<slist.size(); i++)
```

Comment:
Enki's coding convention is to put a space between `if` and the opening parenthesis: `if (`.